app/assets/javascripts/samples/sample_datatable.js

```diff
@@ -11,16 +11,7 @@ table = $("#samples").DataTable({
     order: [[2, "desc"]],
     dom: "RB<'row'<'col-sm-9-custom toolbar'l><'col-sm-3-custom'f>>tpi",
     stateSave: true,
-    buttons: [{
-        extend: "colvis",
-        text: function () {
-            return '<span class="glyphicon glyphicon-option-horizontal"></span> ' +
-                "<span class='hidden-xs-custom'>" +
-                I18n.t('samples.column_visibility') +
-                '</span>';
-        },
-        columns: ":gt(2)"
-    }],
+    buttons: [{}],
```

Comment:
As we don't use built in buttons, we can remove it from here, also removing 'B' from 'dom'